### ISSUE TITLE
FlxUICheckbox Gamepad support?

### ISSUE THREAD

**Issue by Fuzzipede** [HaxeFlixel/flixel-ui#172]
I cannot check checkboxes with my gamepad? Mouse works well. Is it not implemented? Will it be implemented?

Thanks for your help. I really like the Flixel-UI kit! :)

**Comment by larsiusprime** [comment]
You can accomplish this with the FlxUICursor. See the FlxUICursor demo in flixel-demos.

**Comment by Fuzzipede** [comment]
The checkbox in the demo doesnt work for me with Keyboard or Gamepad. Am i missing something?

**Comment by larsiusprime** [comment]
To be clear, we're talking about this demo:
http://haxeflixel.com/demos/Cursor/

Do you have issues with that demo?

**Comment by Fuzzipede** [comment]
yeah. Thats the demo i was trying. I get the  cursor_down when i press the a button on my XBOX360-Pad/Keyboard, but the checkbox does not check or uncheck itself. The buttons working. but not the checkboxes

**Comment by larsiusprime** [comment]
We'll look into this and see if we can reproduce/fix it.

**Comment by Fuzzipede** [comment]
any news about this. Do you need more information from me? 

Thanks for your help!
